src/index.ts

```diff
@@ -86,11 +87,14 @@ function mergeWithStrategy(
 
 function readConfigFile(path: string, type: 'json' | 'yaml'): any {
 	log(`Load extension ${path}`);
+
 	switch (type) {
 		case 'json':
 			return fs.readJSONSync(path);
 		case 'yaml':
 			return JsYaml.load(fs.readFileSync(path, 'utf8'));
+		default:
+			throw new Error(`Invalid type "${type}", can't read file ${path}`);
```

Comment:
Is there a test case on that case ?

Reply:
Nope, eslint require to add this default behavior for the switch instruction. May i have to add a test for that ?

Reply:
yes,  could you add a test for that case ?

Reply:
Regarding the source code, i can't really add a test for that, because the private method readConfigFile is called by  loadExtendConfig, and loadExtendConfig check first if the extention file is correct, yml, yaml or json, if not, an error is raised.
so, the default value from the switch in readConfigFile cannot be targeted. Perhaps it's the reason why there was no default error in the switch.